### ISSUE TITLE
[BUG] Adaptive Cards associatedInputs not working as expected 

### ISSUE THREAD

**Issue by joschi27** [MicrosoftDocs/msteams-docs#4862]
Good day,

TLDR; Adaptive cards in Teams ignore the associatedInputs="None" attribute in adaptive cards with Action.Execute

## Issue Description:
The issue is with the adaptive card associatedInputs feature. It does not work as expected in teams. It seems to completely ignore the associatedInputs attribute.

I only tested this with the windows desktop client.
I have not tested this with the Input.Submit or any other types of actions.

## How to reproduce:
1. Send the JSON (at the end) as an adaptive card to a teams channel
2. Click on the "negative" button.

### Expected Behaviour
The card should be sent, no matter if something is entered in the pinCode-field.
This expected behaviour also works in the adaptivecards.io/designer

### Actual Behaviour
The card is not sent and the errorMessage is displayed under the input field.

### Workaround
What we will do for now is just sending the card in any case, not using the associatedInputs attribute, errorMessage attribute or isRequired attribute and validate serverside, then send back the card with an explicit errorMessage attached to it. This increases traffic on all sides, but works.

Json to send: 
```
{
    "type": "AdaptiveCard",
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.4",
    "body": [
        {
            "type": "Container",
            "containerStyle": "attention",
            "spacing": "none",
            "items": [
                {
                    "type": "ColumnSet",
                    "columns": [
                        {
                            "type": "Column",
                            "items": [
                                {
                                    "type": "TextBlock",
                                    "size": "large",
                                    "text": "Test Page",
                                    "wrap": true
                                },
                                {
                                    "type": "TextBlock",
                                    "spacing": "small",
                                    "isSubtle": true,
                                    "wrap": true,
                                    "text": "Triggered by the novaalert teams-bot"
                                }
                            ]
                        }
                    ]
                }
            ]
        },
        {
            "type": "Container",
            "spacing": "medium",
            "items": [
                {
                    "type": "TextBlock",
                    "text": "PIN"
                },
                {
                    "type": "Input.Text",
                    "placeholder": "enter pin code",
                    "id": "pinCode",
                    "isMasked": true,
                    "style": "password",
                    "isRequired": true,
                    "errorMessage": "Please enter the correct PIN."
                }
            ]
        }
    ],
    "actions": [
        {
            "type": "Action.Execute",
            "fallback": "Action.Submit",
            "title": "Confirm positive",
            "data": {
                "value": "respond_positive"
            },
            "style": "positive",
            "associatedInputs": "Auto"
        },
        {
            "type": "Action.Execute",
            "fallback": "Action.Submit",
            "title": "Confirm negative",
            "data": {
                "value": "respond_negative"
            },
            "style": "destructive",
            "associatedInputs": "None"
        }
    ]
}
```

**Comment by joschi27** [comment]
This is a symptom of https://github.com/MicrosoftDocs/msteams-docs/issues/4862

Closing.
